Modules/_sqlite/module.c

```diff
@@ -391,6 +391,21 @@ static int add_integer_constants(PyObject *module) {
     ret += PyModule_AddIntMacro(module, SQLITE_SAVEPOINT);
 #if SQLITE_VERSION_NUMBER >= 3008003
     ret += PyModule_AddIntMacro(module, SQLITE_RECURSIVE);
+#endif
+    // Run-time limit categories
```

Comment:
In what version they are defined?

Reply:
Is it worth to expose also SQLITE_MAX_*name* constants?

Reply:
> Is it worth to expose also SQLITE_MAX_name constants?

Unfortunately, they are not available to us in `sqlite3.h`.